plasma_framework/test/src/exits/payment/PaymentProcessInFlightExit.test.js

```diff
@@ -429,17 +404,46 @@ contract('PaymentInFlightExitRouter', ([_, ifeBondOwner, inputOwner1, inputOwner
             });
         });
 
-        describe('When the exit is non canonical, and inputs are piggybacked', () => {
+        describe('When any input is spent, given the challenge game result is canonical', () => {
```

Comment:
I understand that's the scenario described in the issue. Right?

Reply:
yup. Also some other tests as put in the description.